File: OfficeManager.Application/UserMasters/Queries/GetUserProfile/UserProfileDto.cs
using OfficeManager.Domain.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OfficeManager.Application.UserMasters.Queries.GetUserProfile
{
    public class UserProfileDto
    {
        public Guid Id { get; set; }
        public string Email { get; set; }
        public string Contact { get; set; }
        public UserRole Role { get; set; }
        public string PersonalEmail { get; set; }
        public DateTime DateOfJoining { get; set; }
    }
}

File: OfficeManagerUI/src/app/shared/Authentication/dtos/UserProfileDto.ts
import { UserRoleDto } from "./UserRoleDto";

export class UserProfileDto{
    id:string;
    email:string;
    contact: string;
    role: UserRoleDto;
    personalEmail: string;
    dateOfJoining: Date;

    constructor(id:string,email:string,contact:string,role:UserRoleDto,personalEmail:string,dateOfJoining:Date){
        this.id = id;
        this.email = email;
        this.contact = contact;
        this.role = role;
        this.personalEmail = personalEmail;
        this.dateOfJoining = dateOfJoining;
    }
}